### PR TITLE
Merge 1.2.10 to dev

### PR DESCRIPTION
**Pull request by ameyapat**
## Proposed changes

Merge release 1.2.10 back into dev

## Type of change

- [ ] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

## Risk

- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [ ] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

### CODE REVIEW

**Review by mipetriu — APPROVED** [review]

**Review by juan-arias — CHANGES_REQUESTED** [review]
Added a question about the IdentityCore merge from master to dev

File: MSAL/IdentityCore
[the remark this answers is not in the document]
Comment (ameyapat) [reply MSAL/IdentityCore:1]: I'm updating the common-core submodule to latest dev on common-core, it includes changes in dev that were merged after release. I didn't want to create another PR to update dev again here with those changes

[the remark this answers is not in the document]
Comment (juan-arias) [reply MSAL/IdentityCore:1]: Oh yeah, it makes sense, thanks

[the remark this answers is not in the document]
Comment (ameyapat) [reply MSAL/IdentityCore:1]: Planning to do the same for msal submodule update on broker

**Pull request merged**